README.md

```diff
@@ -82,7 +82,10 @@ You can check your version of node by running
 node -v
 ```
 
-then install [Yarn](https://yarnpkg.com/lang/en/docs/install/) for your system
+_Currently, Yarn global doesn't work on Windows - so feel free to skip installing Yarn if you're running on Windows_
```

Comment:
Perhaps: "All global installs below are using `npm` over `yarn` because of [this bug](https://github.com/yarnpkg/yarn/issues/859) on Windows, which is known for global installs with Yarn."